build/run-test-kind-prow.sh

```diff
@@ -37,7 +37,7 @@ VERSION_TAG="latest"
 if [ "${PULL_BASE_REF}" ] && [ "${PULL_BASE_REF}" != "main" ]; then
   VERSION_TAG="${VERSION_TAG}-${PULL_BASE_REF#*-}"
 fi
-KIND_COMMAND="cd ${WORK_DIR} && GOROOT=/tmp/go PATH=\$GOROOT/bin:\$PATH deployOnHub=${deployOnHub} RELEASE_BRANCH=${PULL_BASE_REF} VERSION_TAG=${VERSION_TAG} CGO_ENABLED=0 ./build/run-e2e-tests.sh"
+KIND_COMMAND="cd ${WORK_DIR} && GOROOT=/tmp/go PATH=\$GOROOT/bin:\$PATH deployOnHub=${deployOnHub} RELEASE_BRANCH=${PULL_BASE_REF} VERSION_TAG=${VERSION_TAG} CGO_ENABLED=0 K8SCLIENT=kubectl ./build/run-e2e-tests.sh"
```

Comment:
(Though it's probably good to test our code against `kubectl`...)

It's suspicious that the `deployOnHub` tests are passing but the other is not. I suspect there's a context issue somewhere since the managed cluster is created second.

Reply:
Couldn't find anything about context issues. My other theory is the parallelism might be interfering? Looks like we use the same policy for `inform` and `enforce`, so maybe create separate policy names for those?

Reply:
This change is because I didn't bother figuring out how to install `oc` differently in Prow. The setup puts it in `/usr/local/bin`, but for some reason that's not in the PATH on prow? But instead of worrying about that, I figured it was easier to just have the prow tests use `kubectl` with this flag.